boards/nordic/nrf52840dk/src/main.rs

```diff
@@ -88,14 +88,18 @@ const UART_TXD: Pin = Pin::P0_06;
 const UART_CTS: Option<Pin> = Some(Pin::P0_07);
 const UART_RXD: Pin = Pin::P0_08;
 
-const SPI_MOSI: Pin = Pin::P0_20;
-const SPI_MISO: Pin = Pin::P0_21;
-const SPI_CLK: Pin = Pin::P0_19;
+const SPI_MOSI: Pin = Pin::P0_29;
+const SPI_MISO: Pin = Pin::P0_30;
+const SPI_CLK: Pin = Pin::P0_26;
```

Comment:
In my case, I couldn't detect SPI on these pins whereas when I matched them to the nRF52 DK default, it worked! Since I had to keep the rest of the changes, I have just reverted these lines for now with that comment.